services/tenant-ui/frontend/src/components/innkeeper/tenants/deleteTenant/ConfirmTenantDeletion.vue

```diff
@@ -54,19 +56,19 @@ const toast = useToast();
 
 async function handleDelete() {
   if (!isTenantNameCorrect.value) {
-    toast.error(
-      'Incorrect tenant name. Please confirm the correct name before deletion.'
-    );
+    toast.error(t('tenants.settings.confirmDeletionIncorrect'));
     return;
   }
-  innkeeperTenantsStore
-    .deleteTenant(props.tenant.tenant_id)
-    .catch((err: string) => {
-      console.log(err);
-      toast.error('Failure: ${err}');
-    });
-
-  emit('success');
-  emit('closed');
+  try {
+    await innkeeperTenantsStore.deleteTenant(props.tenant.tenant_id);
+    toast.success(
+      t('tenants.settings.confirmDeletionSuccess', [props.tenant.tenant_name])
+    );
+    emit('success');
+    emit('closed');
+  } catch (err: any) {
+    console.log(err);
```

Comment:
Could probably use console.error here

Reply:
Ah yeah, looks like there's a few around that are log in catch block that could be better as error. I'll bulk update them in a next PR.